### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by ktpttl** [Nekoraru22/Valorant-instalock#3]
After I have configured everything correctly when I enter agent select screen nothing happens and I don't know why

**Comment by Nekoraru22** [comment]
Apparently if you don't use the game in full screen it doesn't work as it should. I will try to make a keyboard shortcut to save the positions. Sorry :(

**Comment by ktpttl** [comment]
ok I tried using it with and without fullscreen and it still doesn't work so there's another issue or I'm doing something wrong

**Comment by RealItsMuffin** [comment]
Yeah same with me i have fullscreen always on , idk how to change language but i also set up to insta lock neon , it says its set but when i open game screen it doesnt work idk if valorant vanguard disables it or what but pls tell me the issue

**Comment by Nekoraru22** [comment]
I will record a video to show how I do it.

**Comment by RealItsMuffin** [comment]
Thank you , also pls show me how to change the language

**Comment by itzyashh** [comment]
> I will record a video to show how I do it.

Yes please I tried but I don't what's wrong

**Comment by itzyashh** [comment]
> Thank you , also pls show me how to change the language

Open main.py file in notepad and change 'es' to 'en'

**Comment by RealItsMuffin** [comment]
> 

do i just edit it from es to en?
then thats easy

**Comment by RealItsMuffin** [comment]
Bruh it doesnt work
it just crashes when i change it to EN

**Comment by itzyashh** [comment]
> Bruh it doesnt work it just crashes when i change it to EN

![Uploading 20221021_112814.jpg…]()

**Comment by itzyashh** [comment]
> Bruh it doesnt work it just crashes when i change it to EN

Make sure you don't have typo type `"en"`

**Comment by RealItsMuffin** [comment]
> > Bruh it doesnt work it just crashes when i change it to EN
> 
> Make sure you don't have typo type `"en"`

>I already have a file names en so it just replaces the other file when i change 'es' to'en' and it crashes so I dont understand what ur trying to say,

**Comment by Nekoraru22** [comment]
@RealItsMuffin You just have to edit the `main.pyw` with:
```py
p = Program("InstaLocker", "en").start()
```
It's working for me...

**Comment by RealItsMuffin** [comment]
uh , im kinda stupid so i dont understand but ill try

**Comment by Astrow25** [comment]
When I run the script, personally, nothing happens ¯\_(ツ)_/¯

**Comment by CrazyChicken334** [comment]
Shitty code wth did you do man

**Comment by Nekoraru22** [comment]
> Shitty code wth did you do man

The difference between you and me is that I am able to program what I need on my own while you criticize the code of others when you don't even know how to use a single language. If you dislike it so much, why don't you improve it and upload it to show us all what you are capable of? Best regards

**Comment by ktpttl** [comment]
Dear god what have I done by submitting that issue

**Comment by CrazyChicken334** [comment]
> > Shitty code wth did you do man
> 
> The difference between you and me is that I am able to program what I need on my own while you criticize the code of others when you don't even know how to use a single language. If you dislike it so much, why don't you improve it and upload it to show us all what you are capable of? Best regards

You are right, sorry I was just upset bc it didn't work..
But one thing, I don't understand your intentions.. How _should_ it work? 2 positions on the screen and when does the program know when it should start clicking these macros?

I apologize for being rude!

**Comment by Nekoraru22** [comment]
The program works by waiting for the pixels assigned to the set champion button to match the pixels on the screen, then executes the steps to click on the chosen champion and return to standby.

The problem is that if the game is in full screen the pixels are a little bit higher than if you have the game in window without borders and that small difference breaks the whole operation.

The other problem is the selection speed, because I was trying to give a simple click on the button but the game did not detect it, so it makes a somewhat peculiar animation.

When I have some free time I will comment the code better and upload a video, the program works for me and several friends playing in full screen.

Thank you for your apologies

**Comment by RealItsMuffin** [comment]
yeah about the language thing , im absolutely stupid so just show it in the video also pls . thank you

**Comment by RealItsMuffin** [comment]
Hey @Nekoraru22 pls lmk when you upload the video , appreciate you doing so much work just for a game

**Comment by RealItsMuffin** [comment]
i know what to do but i cant find the main.pyw file and i dont know how to edit it after downloading plus im just not that good in coding

**Comment by xxGalaxyRobotxx** [comment]
> I will record a video to show how I do it.

where is the video i want to know ho to set it up HELP
